graphs/possible_bipartition.py

```diff
@@ -5,8 +5,57 @@ def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n)
```

Comment:
⏱ Time complexity will actually be O(N + E) where N is the number of nodes/dogs in `dislikes` and E is the number of `edges` or dislikes between two dogs. You loop through all of the nodes, then each of the nodes' edges.